SECEdgar/filings/filing_types.py

```diff
@@ -0,0 +1,138 @@
+from enum import Enum
+
+
+class FilingType(Enum):
```

Comment:
What would we change here? We need to have some sort of string to be used in the final URL. Would we want to use static typing across the entire project?

Reply:
Nothing to change here. I was trying to give you kudos for using Enums. With enums, later if we add type checker, this will make our code more robust.